src/app.js

```diff
@@ -1 +1,22 @@
 'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const [fileName, newFileName] = process.argv.slice(2);
```

Comment:
Consider adding input validation to check if both 'fileName' and 'newFileName' are provided.